internal/proxy/testdata/TestApply/golden/no_options_set/etc/environment.d/99ubuntu-proxy.conf

```diff
@@ -0,0 +1 @@
+### This file was generated by ubuntu-proxy-manager - manual changes will be overwritten
```

Comment:
I would change the logic with "no options set" -> no file (and thus, removing file if a previous was around). Then, we have 2 use test cases:
- one with the file around before deleting it
- one with new file and still none after applying "nothing"